### ISSUE TITLE
[Feature Request] Archived/Grouped Workspaces

### ISSUE THREAD

**Issue by seanghay** [getinsomnia/insomnia#2739]
**Is your feature request related to a problem? Please describe.**
When we have a lot of workspaces and some of them are old and we want to archive it because we stopped working on those workspaces, so it would be nice if we can archive them to another place instead of listing them all in the main place. It gets really messy when we got too many workspaces.

I don't really want to delete those unused workspaces because I might need it later.

**Describe the solution you'd like**
One solution that came up to my mind is to have a menu where we can move those workspaces into like the screenshot below

![image](https://user-images.githubusercontent.com/15277233/96398898-2c22bb00-11f7-11eb-8ef3-10c5e6af9c68.png)

**Describe alternatives you've considered**

Also, I have thought of the idea of Grouped Workspaces as well the same as grouped requests, it could also solve this kind of issue.

Being able to group workspaces is a nice feature to have, I think.

**Comment by nijikokun** [comment]
Welcome @seanghay,

This is something that aligns with some of the work we've made on Designer, and something we have been thinking about from a product perspective - bringing the dashboard to Core. The main reason is that the needs of Core in relation to workspace management is similar to needs with the documents, and its getting increasingly hard to provide easy to use, and understandable ways without complex workflows and duplicating patterns in two different ways. This would allow us to have a single unified pattern and an intuitive management for workspaces given it's a common pattern across multiple platforms and applications. 

Bringing this up since we did have plans on doing this after adding grouping to the dashboard which would allow for this functionality.

Given that, will need to have a discussion on whether we want to approach this as a stop-gap solution that would go away and how it's migration path would look.
